docs/auth-provider-setup/byo-jwt-provider/jose.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Create JWT using jose package
+sidebar_label: JWT using jose
```

Comment:
This sidebar label looks repeated. Just having jose, jsonwebtoken, passport-jwt etc will look better.

Reply:
Done